content/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM node:12.18.3
+
+WORKDIR /usr/src/app/webapp
+
+RUN apt-get update && apt-get install -y python python-pip rename
+RUN pip install awscli
+
+ADD ./.flowconfig ./.flowconfig
+ADD ./package.json ./package.json
+ADD ./yarn.lock ./yarn.lock
+ADD ./common ./common
```

Comment:
Because the context is set in the docker-compose, I wonder if this is confusing having this file here? Maybe we could have them in a `dockerfiles` folder in the root? Or have a note mentioning that this is the case, and this file wont' actually work outside of the context set in the docker-compose?

Reply:
I went back and forth on this - I think they should stand alone without compose. I think I may be able to simply refer to files in the directory above, so then the context would be this dir. That would make all the images build the same way without changing context.

TBH I have struggled to find a good way to share the "common" context in docker (and as you see a few dockerfiles have identical starting lines).

Reply:
🧠 💨  You can't jump out of the context set by the docker-compose.yml. I'm going to experiment a bit, but will probably end up adding a comment if I can't get the build to work another way.

Reply:
I tried working on how we might do this and setting the context is the only way I could find.

It would be good to know if you find something as the issue with this is the context becomes very very large because of `node_modules` - but maybe that is just something we need to do.